applications/spark/tableau.md

```diff
@@ -0,0 +1,135 @@
+# Visualize Data with Tableau
```

Comment:
This functionality could be useful to people that do not normally run Apache Spark. If there is value, more could be done to simplify the process of running these scripts. It could be a managed environment, for example (module load apache-spark). Please let me know if there is interest in this. Or if you want a demo.

Reply:
@daniel-thom Sure, we could set up a quick demo and go from there